scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -483,7 +493,7 @@ public void onClick(MenuItemClickEvent event) {
             }
         });
 
-        schedKillButton = new MenuItem("Kill");
+        schedKillButton = new MenuItem(KILL);
         schedKillButton.setIcon(SchedulerImages.instance.scheduler_kill_16().getSafeUri().asString());
         schedKillButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)